tac/project-updates/2025/2025-annual-Identus.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: default
+title: 2025 Annual Review Identus
+parent: 2025
+grand_parent: Project Updates
+---
+
+## Project Health
+
+LF Decentralized Trust Identus [LFX Insights Dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/reports/contributors?project=identus&bestPractice=false&dateFilters=Last%20Year&dateRange=2024-01-01%20to%202024-12-31&compare=PP&granularity=month&hideBots=true&repository=all)
+The project is relatively new to the LF Decentralized Trust Ecosystem and is mainly supported by the IOG maintainers.
+At the same time, over the past year, we have seen several new contributors and evidence of usage of the Identus platform by other companies: 
+- https://socious.io/, Socious production use case.
+- https://ahau.io/, Ahau PoC.
+- https://blocktrust.dev/, - identity wallet.
+
+We currently utilize two Discord servers: the LF Decentralized Trust Discord and the private Identus Discord server. 
+The Identus Discord has seen significantly more engagement. 
```

Comment:
Just to clarify, what do you mean by "private"?

Reply:
"dedicated" its a better word. The discord it's public it was created before we joined hyperledger.

https://discord.gg/mKqBT3vaXh